src/annotator/components/NotebookModal.js

```diff
@@ -53,9 +55,7 @@ export default function NotebookModal({ eventBus, config }) {
   const [isHidden, setIsHidden] = useState(true);
   const [groupId, setGroupId] = useState(/** @type {string|null} */ (null));
   const originalDocumentOverflowStyle = useRef('');
-  const emitter = useRef(
-    /** @type {ReturnType<eventBus['createEmitter']>|null} */ (null)
-  );
+  const emitterRef = /** @type {{ current: Emitter|null }} */ (useRef(null));
```

Comment:
Maybe trimming start and end whitespaces?
```suggestion
  const emitterRef = /** @type {{current: Emitter|null}} */ (useRef(null));
```

Reply:
I don't have a hugely strong aesthetic preference here, but I used whitespace because that's how Prettier formats a type in TS (if you do `type Ref = {current: HTMLElement}`) and also how the majority of our current inline object types are formatted.

Reply:
OK, sure.